conf/evolutions/default/94.sql

```diff
@@ -0,0 +1,10 @@
+# --- !Ups
+ALTER TABLE virtual_challenges DROP CONSTRAINT IF EXISTS CON_VIRTUAL_CHALLENGES_USER_ID_NAME;
```

Comment:
Add a comment to the file describing the what/why this this is needed. Something like `-- Remove the constraint for a distinct virtual challenge (id,name) tuple; allow users to create virtual challenges with the same name.`